CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ set(ROCMLIR_DRIVER_MISC_E2E_TEST_ENABLED 0 CACHE BOOL "Enable build miscellaneou
 set(ROCMLIR_DRIVER_PR_E2E_TEST_ENABLED 1 CACHE BOOL "Enable build PR-triggered E2E tests for Rock driver")
 set(ROCMLIR_DRIVER_RANDOM_DATA_SEED "none" CACHE STRING "Enable E2E tests using random data")
 set(ROCMLIR_DRIVER_XDLOPS_TEST_ENABLED 0 CACHE BOOL "Enable build E2E tests for XDLops")
+set(ROCMLIR_DRIVER_FEATURE "mfma" CACHE BOOL "Choose the feature")
```

Comment:
These should be defaulted to empty string (or "none" or what have you) and the booleans below should default to 0. The build should work by default on a non-xdlops scenario